pkg/utils/utils.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() error {
+	errMsg := ""
+	registriesConfPath := sysregistries.RegistriesConfPath(nil)
+	_, err := os.Stat(registriesConfPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			errMsg = fmt.Sprintf("Required file %s not found.\n", registriesConfPath)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s, %s", registriesConfPath, err)
+		}
+	}
+
+	storageDefaultPath := "/etc/containers/storage.conf"
+	_, err = os.Stat(storageDefaultPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			errMsg = fmt.Sprintf("Required file %s not found.\n %s", storageDefaultPath, errMsg)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s, %s.\n %s", storageDefaultPath, err, errMsg)
+		}
+	}
```

Comment:
The code in `c/storage.ReloadConfigurationFile` explicitly supports the situation when the file is missing. Is Buildah doing something special that makes it different from other users that it requires a config file? (E.g. if it requires some non-default configuration option, that option should either become a default, or this code should check that that option is set as necessary.)